lib/core/src/Cardano/Wallet/Primitive/Types/TokenBundle.hs

```diff
@@ -293,6 +294,13 @@ add (TokenBundle (Coin c1) m1) (TokenBundle (Coin c2) m2) =
 subtract :: TokenBundle -> TokenBundle -> Maybe TokenBundle
 subtract a b = guard (b `leq` a) $> unsafeSubtract a b
 
+difference :: TokenBundle -> TokenBundle -> TokenBundle
```

Comment:
Should describe the difference compared to `subtract` — which I'm, to be honest, not entirely sure of.

Reply:
Updated. Although, not with emphasis on the difference to `subtract`.